x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/engines/engines_list_flyout_logic.test.ts

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { LogicMounter } from '../../../__mocks__/kea_logic';
+
+import { nextTick } from '@kbn/test-jest-helpers';
+
+import { Status } from '../../../../../common/types/api';
+import { EnterpriseSearchEngineDetails } from '../../../../../common/types/engines';
+
+import { FetchEngineApiLogic } from '../../api/engines/fetch_engine_api_logic';
+
+import { EngineListFlyoutValues, EnginesListFlyoutLogic } from './engines_list_flyout_logic';
+
+const DEFAULT_VALUES: EngineListFlyoutValues = {
+  fetchEngineData: undefined,
+  fetchEngineName: null,
+  isFetchEngineFlyoutVisible: false,
+  fetchEngineApiStatus: Status.IDLE,
+  fetchEngineApiError: undefined,
+  isFetchEngineLoading: false,
+};
+const mockEngineData: EnterpriseSearchEngineDetails = {
```

Comment:
Got these mockEngineData from [x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/engine/engine_indices_logic.test.ts](https://github.com/elastic/kibana/blob/main/x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/engine/engine_indices_logic.test.ts#L20)

Wasn't sure if have to move `mockEngineData` variable to some common place or may be call directly from the `engine_indices_logic.test.ts` file